ops/model.py

```diff
@@ -1790,6 +1790,14 @@ class UnknownStatus(StatusBase):
     A unit-agent has finished calling install, config-changed and start, but the
     charm has not called status-set yet.
 
+    This status is for READ ONLY and should not be used to set a unit status. The following
+    charm code would be an invalid charm code.
+
+        ```
+        # Raises ops.model.ModelError : ERROR invalid status "unknown", expected one of
+        # [maintenance blocked waiting active]
+        self.unit.status = UnknownStatus()
+        ```
```

Comment:
I think this is probably saying too much, and we don't need an example of what *not* to do. Perhaps just this:
```suggestion
    This status is read-only; trying to set unit or application status to
    ``UnknownStatus`` will raise :class:`ModelError`.
```

Reply:
Simplified, thank you for the suggestion!